subprojects/rapidjson.wrap

```diff
@@ -4,3 +4,7 @@ source_url = https://github.com/Tencent/rapidjson/archive/v1.1.0.tar.gz
 source_filename = rapidjson-1.1.0.tar.gz
 source_hash = bf7ced29704a1e696fbccf2a2b4ea068e7774fa37f6d7dd4039d0787f8bed98e
 patch_directory = rapidjson
+
+[provide]
+rapidjson = rapidjson_dep
+
```

Comment:
extra newline not needed and recommended against.

Reply:
done